src/main/scala/Dao/DAO.scala

```diff
@@ -3,14 +3,13 @@ package Dao
 import java.util.UUID
 import scala.collection.mutable.ListBuffer
 import models.Users
-
 import scala.concurrent.Future
 
 trait DAO {
 
   def addUser(user: Users): Future[String]
 
-  def getById(userId: UUID): Future[Option[Users]]
+  def getById(userId: UUID): Future[ListBuffer[Users]]
```

Comment:
Why is this is a `Future[ListBuffer[Users]]` and not `Future[Option[Users]]`

Reply:
Actually, I was getting some issues with the Future[Option[Users]].
when I was returning Future[Option[Users]], I was getting an issue with the returning values, Like when it was returning
 Some(value) then I could return value but in the case of None, I was confused about what to return. 
That's why I changed it to Future[ListBuffer[Users]]]